azext_iot/central/_help.py

```diff
@@ -520,6 +661,23 @@ def _load_central_command_help():
                 --device-id {deviceid}
                 --interface-id {interfaceid}
```

Comment:
Double check on --interface-id, is this a legacy meaning of combined command path? be sure it does not have conflict with what we provided from --module-name and --component-name.

Reply:
No conflicts now, if interface is is component name, we'll treat it as component name. Otherwise we ignore it.

Reply:
I need to check the interface in device module.

Reply:
Add one more check for interface in device module, if interface is component in device module, it should be treated as component name into api call as well.